backend/src/api/model/search/mod.rs

```diff
@@ -186,9 +230,48 @@ pub(crate) async fn perform(
     // Perform the searches
     let res = tokio::try_join!(
         event_query.execute::<search::Event>(),
+        series_query.execute::<search::Series>(),
         realm_query.execute::<search::Realm>(),
     );
-    let (event_results, realm_results) = handle_search_result!(res, SearchOutcome);
+    let (event_results, series_results, realm_results) = handle_search_result!(res, SearchOutcome);
+
+    // Get thumbnails for series
+    let ids = series_results.hits.iter()
+        .map(|r| r.result.id.0)
+        .collect::<Vec<_>>();
+
+    let (selection, mapping) = select!(series, is_live, audio_only, thumbnail);
+    let thumbnail_info_query = format!("\
+        select {selection} \
+        from ( \
+            select e.series, e.is_live, e.thumbnail, \
+                not exists ( \
+                    select 1 from unnest(e.tracks) as t where t.resolution is not null \
+                ) as audio_only, \
+                row_number() over ( \
+                    partition by e.series order by e.created desc \
+                ) as row_num \
+            from events e \
+            where e.series = any($1) and (read_roles || 'ROLE_ADMIN'::text) && $2 \
+            and (e.start_time <= current_timestamp or e.start_time is null) \
```

Comment:
I wasn't aware that this should filter out already ended live events. I meant to filter out upcoming events with this. Also, how would `start_time <= current_timestamp` filter out currently live events? Or am I misunderstanding your comment?

Reply:
Oh woops, got the logic twisted in my brain. Yeah this does indeed filter out upcomings.

Why though?

Regarding filtering out ended live events: we currently do that in series blocks (see `isPastLiveEvent`). Ended live events are not useful for the user and are later replaced with a recording of the live event. I suppose it also make sense to filter them out here as otherwise people see these in the 3-image-preview but can't see them when clicking on the series? :thinking:

Reply:
Ah ye ok fair. Don't remember :see_no_evil: